src/components/VFilters/VFilterChecklist.vue

```diff
@@ -20,40 +20,55 @@
         <VLicense v-if="filterType === 'licenses'" :license="item.code" />
         <template v-else>{{ itemLabel(item) }}</template>
       </VCheckbox>
-      <button
-        v-if="filterType === 'licenses'"
-        :ref="`${item.code}licenseIcon`"
-        :aria-label="$t('browse-page.aria.license-explanation')"
-        class="text-dark-charcoal-70 appearance-none"
-        type="button"
-        @click.stop="toggleLicenseExplanationVisibility(item.code)"
-      >
-        <VIcon :icon-path="helpIcon" />
-      </button>
+
+      <!-- License explanation -->
+      <VPopover v-if="filterType === 'licenses'">
+        <template #trigger="{ a11yProps }">
+          <VButton
+            v-bind="a11yProps"
+            variant="plain"
+            size="disabled"
+            :aria-label="$t('browse-page.aria.license-explanation')"
+            class="text-dark-charcoal-70"
+            type="button"
+          >
+            <VIcon :icon-path="icons.help" />
+          </VButton>
+        </template>
+        <template #default="{ close }">
+          <div class="relative">
+            <VIconButton
```

Comment:
This button needs an aria label describing it as a close button.

Reply:
Added ARIA-label in [3f91c0e](https://github.com/WordPress/openverse-frontend/pull/850/commits/3f91c0e3354840c516c7026d2d8ee03e8ba99ae3).